src/components/LightIntensity/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import { LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, Legend } from 'recharts';
+import { Stack } from '@mui/material';
+import { PLUGIN_KEYS } from '../../constants';
+import usePluginState from '../../hooks/usePluginState';
+import SensorCard from './SensorCard';
+import DeviceThermostatIcon from '@mui/icons-material/DeviceThermostat';
+import OpacityIcon from '@mui/icons-material/Opacity';
+import AirIcon from '@mui/icons-material/Air';
+import { blue, purple, red } from '@mui/material/colors';
+import useMessage from '../../hooks/useMessage';
+
+
+const LightIntensity = () => {
+    const { message } = useMessage(PLUGIN_KEYS.SENSOR_RAW, '/sensor_raw', 500);
```

Comment:
Use of duplicate useMessage key. Instead of using useMessage hook, we can use usePluginState to get data from the redux store. If we need to use data from the same ROS topic in multiple plugins, we might consider implementing a new hook or changing the design slightly.